comp/otelcol/collector/impl/testdata/simple-dd/config-enhanced-result.yaml

```diff
@@ -1,204 +0,0 @@
-connectors: {}
```

Comment:
These 3 yaml files do not seem to be used anywhere, probably were missed out when moving code around.